common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -253,24 +253,27 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           </h3>
         </div>
 
-        {this._renderContactHackathon()}
+        
 
-        <div className="corporate-hackathon-saying corporate-bg-light col-12">
+        <div className="corporate-hackathon-saying col-12 corporate-bg-light">
           <h1>What our partners are saying.</h1>
           <h2>
             Feedback from our partners about their custom hackathon events.
           </h2>
-          <div className="carousel-testimonial-root">
+          <div className="carousel-testimonial-root corporate-bg-light">
             <TestimonialCarousel
               category="partner-highlights"
               interval={600000}
             />
           </div>
           <p className="h3">
-            Read about previous events in our <a href={window.BLOG_URL}>blog</a>
-            .
+            {/* TODO find where {window.BLOG_URL} is declared */}
```

Comment:
Let's remove this and I'll write an issue that we need to document environment variables better in our Contributor's Guide. I reviewed it this morning and agree that we need a section about how to use them, not just how to load them.

Reply:
I created issue #895 for this one. If there's any specifics you want to make sure we include in the documentation, please comment on it there.